packages/office-ui-fabric-react/src/components/List/docs/ListOverview.md

```diff
@@ -3,3 +3,5 @@ List provides a base component for rendering large sets of items. It is agnostic
 **Performance is important, and DOM content is expensive. Therefore limit what you render.** Unlike a simple for loop that renders all items in a set, a List uses ui virtualization. It only renders a subset of items, and as you scroll around, the subset of rendered content is shifted to what you're looking at. This gives a much better experience for large sets, especially when the per-item components are complex/render intensive/network intensive.
 
 Lists break down the set of items passed in into pages. Only pages within a "materialized window" are actually rendered. As that window changes due to scroll events, pages that fall outside that window are removed, and their layout space is remembered and pushed into spacer elements. This gives the user the experience of browsing massive amounts of content but only using a small number of actual elements. This gives the browser much less layout to resolve, and gives React DOM diffing much less content to worry about.
+
+**Note: although onRenderCell is an optional member of IListProps, if you opt not to use an onRender() method, you need to format each element of your IList's props *items* value using the following format: {[name: *insert data here*]}.** An example array passed to items would be {[name: item1], [name: item2]}
```

Comment:
where you're referencing an actual Typescript type, use backticks so the code is tagged `like this`

Reply:
Ok. so update to ``{[name: item1], [name: item2]}`` ?

Reply:
yeah with the backticks

Reply:
`
**Note: although `onRenderCell` is an optional member of `IListProps`, if you opt not to use an `onRender()` method, you need to format each element of your `IList`'s props *items* value using the following format: `{[name: *insert data here*]}`.** An example array passed to items would be `[{name: item1}, {name: item2}]`
`
By the way, that last example array was wrong, just a typo.